ui/lib/services/bom_bar_client.dart

```diff
@@ -38,9 +38,10 @@ class BomBarClient {
     ));
   }
 
-  /// Returns all available projects.
-  Future<List<Project>> getProjects() async {
-    final response = await dio.getUri(_projectsUrl);
+  /// Returns all projects or projects matching the provided [fragment] fragment.
```

Comment:
The last "fragment" can be dropped, as the generated help nicely keeps the `[fragment]` text in the description.